models/yang/sonic/sonic-tam.yang

```diff
@@ -0,0 +1,69 @@
+module sonic-tam {
+    namespace "http://github.com/Azure/sonic-tam";
+    prefix tam;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC TAM";
+
+    revision 2019-11-08 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-tam {
+
+        container TAM_DEVICE_TABLE {
+
+            list TAM_DEVICE_TABLE_LIST {
+                key "device";
```

Comment:
Change this to 'name' instead of 'device'.

Reply:
Done